src/main/java/org/jenkins_ci/plugins/flexible_publish/DefaultPublisherDescriptorLister.java

```diff
@@ -39,7 +39,7 @@
 
 public class DefaultPublisherDescriptorLister implements PublisherDescriptorLister {
 
-    public static final List<String> EXCLUSIONS = Arrays.asList(
+    private static final List<String> EXCLUSIONS = Arrays.asList(
```

Comment:
<BugInstance type='MS_MUTABLE_COLLECTION_PKGPROTECT' priority='Normal' category='MALICIOUS_CODE' message='org.jenkins_ci.plugins.flexible_publish.DefaultPublisherDescriptorLister.EXCLUSIONS is a mutable collection which should be package protected' lineNumber='42'/>

I decided to make this private even it causes breaking binary compatibility.
This field doesn't look accessed from other plugin (at least official plugins hosted in jenkinsci)